snakemake_executor_plugin_slurm/__init__.py

```diff
@@ -88,6 +90,10 @@ def run_job(self, job: JobExecutorInterface):
         call += self.get_account_arg(job)
         call += self.get_partition_arg(job)
 
+        if job.resources.get("clusters"):
+            call += f" --clusters {job.resources.clusters}"
+            self._clusters = job.resources.clusters
```

Comment:
What happens if there are multiple jobs that select different clusters? To me this seems like that the check_active_jobs command will always take the clusters setting from the latest submitted job, which can't be correct in the general case.

Reply:
Right. While SLURM itself supports submitting to multiple clusters, with the current scheme we would need to record a list or strictly speaking, a set of clusters, which we constantly query: i.e. if cluster 'A' is selected for job 'a' and cluster 'B' for job 'b', there status query has to go to 'A,B'.

Would that constitute an acceptable solution?

Reply:
sacct also accepts `--clusters all` this might be a convenient solution, maybe slightly less precise.

Reply:
Thank you, easy enough! This went into 67cfc6c286564b710402d64ad679acbd378758a7 .

However, we do not have a test for it — yet.